### PR TITLE
fix dcos-log integration tests

### PR DESCRIPTION
**Pull request by mnaboka**
blocked by
  - [dcos-log](https://github.com/dcos/dcos-log/pull/47)
  - [oss dcos](https://github.com/dcos/dcos/pull/2183)
  - [ee dcos](https://github.com/mesosphere/dcos-enterprise/pull/1854)
  - [oss dcos](https://github.com/dcos/dcos/pull/2224)

### CODE REVIEW

**Comment by mnaboka** [comment]
run all integration tests

File: cli/tests/integrations/test_task.py [hunk 1]
```diff
@@ -100,7 +104,6 @@ def test_filter():
 
 # TODO (Bilal): commenting log tests as they are currently failing with
 # DC/OS master. Maksym will take a look at these before the 1.11 release.
```
Comment (bamarni) [inline cli/tests/integrations/test_task.py]: Can you also remove the comment here?

**Comment by mnaboka** [comment]
looks like some genconf errors
```
[linux-tests] [1;31mexhibitor_storage_backend: Must set exhibitor_storage_backend, no way to calculate value.[0m
[linux-tests] [919] Failed to execute script cli
[linux-tests] Traceback (most recent call last):
[linux-tests]   File "dcos_launch/cli.py", line 133, in <module>
[linux-tests]   File "dcos_launch/cli.py", line 125, in main
[linux-tests]   File "dcos_launch/cli.py", line 91, in do_main
[linux-tests]   File "dcos_launch/onprem.py", line 177, in wait
[linux-tests]   File "dcos_launch/platforms/onprem.py", line 129, in do_genconf
[linux-tests]   File "site-packages/dcos_test_utils/ssh_client.py", line 52, in command
[linux-tests]   File "subprocess.py", line 626, in check_output
[linux-tests]   File "subprocess.py", line 708, in run
[linux-tests] subprocess.CalledProcessError: Command '['/usr/bin/ssh', '-oConnectTimeout=10', '-oStrictHostKeyChecking=no', '-oUserKnownHostsFile=/dev/null', '-oLogLevel=ERROR', '-oBatchMode=yes', '-oPasswordAuthentication=no', '-oControlPath=/tmp/tmp9u1d8s4v', '-oControlMaster=auto', '-p', '22', 'centos@34.210.138.206', 'sudo', 'bash', '/home/centos/dcos_generate_config.sh', '--genconf']' returned non-zero exit status 1
```

**Comment by mnaboka** [comment]
run all integration tests

**Comment by mnaboka** [comment]
run all integration tests

**Comment by mnaboka** [comment]
run all integration tests

**Comment by mnaboka** [comment]
run all integration tests

**Comment by mnaboka** [comment]
run all integration tests

**Comment by mnaboka** [comment]
run all integration tests

**Pull request merged**